corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -217,6 +217,29 @@ def toggle_enabled(request, toggle_or_toggle_name):
     import corehq.toggles
     return _toggle_enabled(corehq.toggles, request, toggle_or_toggle_name)
 
+
+def _ui_notify_enabled(module, request, ui_notify_instance_or_name):
+    ui_notify = _get_obj_from_name_or_instance(module, ui_notify_instance_or_name)
+    return ui_notify.enabled(request)
+
+
+@register.filter
+def ui_notify_enabled(request, ui_notify_instance_or_name):
+    import corehq.apps.notifications.ui_notify
+    return _ui_notify_enabled(
+        corehq.apps.notifications.ui_notify,
+        request,
+        ui_notify_instance_or_name
+    )
+
+
+@register.filter
+def ui_notify_slug(ui_notify_instance_or_name):
```

Comment:
the name is the name of the variable in ui_toggles, but the slug is what's actually saved in the DB. I think the two should be separated. maybe we want to keep a shorthand name, but give a really long slug with more info `app_builder_publish_08_2017_v001`